### ISSUE TITLE
Consider changing `jib:build

### ISSUE THREAD

**Issue by coollog** [GoogleContainerTools/jib#327]
**Description of the issue**:

**Expected behavior**:

**Steps to reproduce**: 

**Environment**: 

**`jib-maven-plugin` Configuration:**
```xml
PASTE YOUR pom.xml CONFIGURATION HERE
```

**Log output**: 

**Additional Information**:

**Comment by TadCordle** [comment]
I'm assuming this is for consistency with the gradle plugin?

I vote that it's better as `jib:build`. `jib:jib` matches the gradle plugin better, but at the cost of reduced internal consistency (`jib:jib` seems out of place next to `jib:dockerContext` and `jib:dockerBuild`), plus `jib:build` is more descriptive about what it does. If anything I vote changing the gradle task to `jibBuild` instead of `jib` if we want the two plugins to be more similar.

**Comment by patflynn** [comment]
I do find jib:jib a little odd. Can you go into more detail about the
rationale?

**Comment by coollog** [comment]
Yeah, this was only a suggestion for consideration. This was mostly as a way to have more consistency between maven and gradle. In Maven, there are other plugins that call its default goal the same name as the plugin itself.
